libcaf_opencl/src/manager.cpp

```diff
@@ -169,12 +161,12 @@ program_ptr manager::create_program(const char* kernel_source,
       // seems that just apple implemented the
       // pfn_notify callback, but we can get
       // the build log
-#ifndef __APPLE__
+#ifndef CAF_MACOS
       CAF_LOG_ERROR(CAF_ARG(ss.str()));
 #endif
       oss << endl << ss.str();
     }
-    CAF_RAISE_ERROR(oss.str());
+    CAF_RAISE_ERROR("clBuildProgram failed");
```

Comment:
If I'm not mistaken `oss` is never read now and should be removed.

Reply:
Fixed.